Fix FAQ link from jumping to database section (#13566)
sveltejs/kit: documentation/docs/60-appendix/20-integrations.md

## Integration FAQs

> The SvelteKit FAQ has a [how to do X with SvelteKit](./faq), which may be helpful if you still have questions.

Reviewer: ```suggestion
[The SvelteKit FAQ](./faq) answers many questions about how to do X with SvelteKit, which may be helpful if you still have questions.
```
